docs_user/modules/proc_using-new-subnet-ranges.adoc

```diff
@@ -1,43 +1,23 @@
 
-The general idea here is to define new IP ranges for control plane services
-that belong to a different subnet that was not used in the existing cluster.
-Then, configure link local IP routing between the old and new subnets to allow
-old and new service deployments to communicate. This involves using {OpenStackPreviousInstaller}
-mechanism on pre-adopted cluster to configure additional link local routes
-there. This will allow EDP deployment to reach out to adopted nodes using their
-old subnet addresses.
+You must size the new subnet appropriately to accommodate the new control
+plane services. There are no specific requirements for the
+existing deployment allocation pools already consumed.
 
-The new subnet should be sized appropriately to accommodate the new control
-plane services, but otherwise doesn't have any specific requirements as to the
-existing deployment allocation pools already consumed. Actually, the
-requirements as to the size of the new subnet are lower than in the second
-scenario, as the old subnet ranges are kept for the adopted nodes, which means
-they don't consume any IP addresses from the new range.
+In the following procedure, you configure `NetworkAttachmentDefinition` custom resources (CRs) to use a different subnet from what is configured in the `NetConfig` CR for the same networks. The former range is used for control plane services, while the new range is used to manage IP Address Management (IPAM) for data plane nodes.
 
-In this scenario, you will configure `NetworkAttachmentDefinition` custom resources (CRs) to use a
-different subnet from what will be configured in `NetConfig` CR for the same
-networks. The former range will be used for control plane services,
-while the latter will be used to manage IPAM for data plane nodes.
+[IMPORTANT]
+Do not change adopted node IP addresses during the adoption process. List the IP addresses in the `fixedIP` fields in the `OpenStackDataPlaneNodeSet` CR `per-node` section.
 
-During the process, you will need to make sure that adopted node IP addresses
-don't change during the adoption process. This is achieved by listing the
-addresses in `fixedIP` fields in `OpenstackDataplaneNodeSet` per-node section.
+.Procedure
 
----
-
-Before proceeding, configure host routes on the adopted nodes for the
-control plane subnets.
-
-To achieve this, you will need to re-run `openstack overcloud node provision` with additional `routes` entries added to `network_config`. (This change should be applied for every adopted node configuration.) For example, you may add the following to `net_config.yaml`:
-
-```yaml
+. In the `NetworkAttachmentDefinition` CR, configure host routes on the adopted nodes for the control plane subnets. For example:
```

Comment:
This is incorrect. The example below is part of TripleO configuration and it demonstrates how to configure static routes there (before adoption proper is started). It is *not* `NetworkAttachmentDefinition`. (TripleO configuration is a custom .yaml file that is not a CR - CR being `Custom Resource` in OpenShift API parlance.)

Reply:
OK, maybe something like the following instead?

"In your network configuration file, configure host routes on the RHOSP nodes for the control plane subnets. For example:"

I'm not sure what the custom yaml file should be called. I guess it varies depending on the customer, but I'm trying to help the user understand which file they are supposed to be updating.

Reply:
It probably can use a different name, but AFAIU the `net_config.yaml` is a common (default) name, so we may mention it as a likely example.

---

Yes, your new wording is better. Thanks.